### PR TITLE
DATA: Model & Migrate Meals

### PR DESCRIPTION
**Pull request by shivangsoni**

### CODE REVIEW

File: OtripleS.Web.Api/Brokers/Storages/IStorageBroker.Meals.cs [hunk 1]
```diff
@@ -0,0 +1,14 @@
+﻿// ---------------------------------------------------------------
+// Copyright (c) Coalition of the Good-Hearted Engineers
+// FREE TO USE AS LONG AS SOFTWARE FUNDS ARE DONATED TO THE POOR
+// ---------------------------------------------------------------
+
```
Comment (ElbekDeveloper) [inline OtripleS.Web.Api/Brokers/Storages/IStorageBroker.Meals.cs:5]: Remove this empty line. Leave only one to separate the header and the code

File: OtripleS.Web.Api/Brokers/Storages/IStorageBroker.Meals.cs [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+
+namespace OtripleS.Web.Api.Brokers.Storages
+{
+    public partial interface IStorageBroker
+    {
+
+
+    }
```
Comment (ElbekDeveloper) [inline OtripleS.Web.Api/Brokers/Storages/IStorageBroker.Meals.cs:13]: slap these together `{ }` like this.

File: OtripleS.Web.Api/Models/Meals/Meal.cs [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace OtripleS.Web.Api.Models.Meals
+{
+    public class Meal
+    {
+        public Guid Id { get; set; }
+        public enum Type { Breakfast, Lunch, Dinner };
```
Comment (ElbekDeveloper) [inline OtripleS.Web.Api/Models/Meals/Meal.cs:16]: Maybe it is better to have  this enum as a separate type

File: OtripleS.Web.Api/Models/Meals/Meal.cs [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+        public double Price { get; set; }
+        public Guid StudentId { get; set; }
+        public enum Status { Ready, Pickedup, Delivered, Cancelled };
```
Comment (ElbekDeveloper) [inline OtripleS.Web.Api/Models/Meals/Meal.cs:19]: Same

File: OtripleS.Web.Api/Models/Meals/Meal.cs [hunk 5]
```diff
@@ -0,0 +1,27 @@
...
+        public int Quantity { get; set; }
+        public Guid CreatedBy { get; set; }
+        public Guid UpdatedBy { get; set; }
+        public DateTimeOffset CreatedDate { get; set; }
+        public DateTimeOffset UpdatedDate { get; set; }
+
```
Comment (ElbekDeveloper) [inline OtripleS.Web.Api/Models/Meals/Meal.cs:25]: Remove this empty line

**Comment by ElbekDeveloper** [comment]
https://github.com/hassanhabib/CSharpCodingStandard generally recommend this book as well. Take care ; )

**Comment by ElbekDeveloper** [comment]
You can commit all these changes as `CODE RUB: Resolve Comments`

File: OtripleS.Web.Api/Brokers/Storages/IStorageBroker.Meals.cs (hunk 1 above)
Comment (shivangsoni) [reply OtripleS.Web.Api/Brokers/Storages/IStorageBroker.Meals.cs:5]: Done

**Review by ElbekDeveloper — APPROVED** [review]
I am not sure about the `AddMealsModel.Designer.cs` file. But the rest is legitimate. I like it

**Pull request merged**